client/src/components/img-button/img-button.js

```diff
@@ -0,0 +1,62 @@
+import Component from '@/components/component';
+
+import ImgBox from '@/components/img-box/img-box';
+import DefaultImg from '@/assets/images/button-default.svg';
+
+import styles from '@/styles/components/img-button/img-button.module.scss';
+
+const MAX_IMAGE_UPLOAD = 10;
+
+export default class ImgButton extends Component {
+  constructor(props) {
+    super(props);
+
+    this.$dom = this.createDom('div', {
+      className: styles['img-button'],
+    });
+
+    this.ImgBox = new ImgBox({
+      type: 'medium',
+      imageURL: '',
+      onClick: () => {},
+    });
+
+    this.render();
+    this.addEvent();
+  }
+
+  render = () => {
+    if (this._props.buttonType === 'add') {
+      this.$dom.innerHTML = `
+        <div class="${styles['ImgBox']}"></div>
+        <div class="${styles['content-wrapper']}">
+          <img src="${DefaultImg}">
+          <div class="${styles['image-count']}">${this._props.imageCount}/${MAX_IMAGE_UPLOAD}</div>
+        </div>
+      `;
+    } else {
```

Comment:
```suggestion
    } else if (this._props.buttonType === 'image') {
```

type이 `add`와 `image` 두개 밖에 없지만 그래도 한번더 검사를 하면 좋겠습니다.
사용자가 실수로 잘못 입력하게되면 그냥 렌더가 되지 않는것이죠.